views/layouts/main.handlebars

```diff
@@ -6,6 +6,16 @@
   <meta name="defaultLanguage" content="en-US">
   <meta name="availableLanguages" content="{{availableLanguages}}">
 
+  <meta property="og:title" content="Firefox Send"/>
+  <meta name="twitter:title" content="Firefox Send"/>
+  <meta name="description" content="Encrypt and send files with a link that automatically expires to ensure your important documents don’t stay online forever."/>
+  <meta property="og:description" content="Encrypt and send files with a link that automatically expires to ensure your important documents don’t stay online forever."/>
+  <meta name="twitter:description" content="Encrypt and send files with a link that automatically expires to ensure your important documents don’t stay online forever."/>
+  <meta name="twitter:card" content="summary"/>
+  <meta property="og:image" content="/resources/send-fb.jpg"/>
+  <meta name="twitter:image" content="/resources/send-twitter.jpg"/>
+  <meta property="og:url" content="https://send.firefox.com"/>
```

Comment:
These last 3 seem almost questionable...

1. Should we be pulling `og:url` from some environment? Like check `conf.env` and set this to dev/stage/prod as appropriate? I'm not 100% certain what the `og:url` is used for.
2. For `og:image` and `twitter:image`, I thought those needed to be fully qualified URLs. I vaguely recall us struggling with this on Test Pilot site. Ref: [**mozilla/testpilot** /frontend/tasks/pages.js:16-19](https://github.com/mozilla/testpilot/blob/23a62b5a91042c46d20a317193332e4b167e1825/frontend/tasks/pages.js#L16-L19)

Although hard coding prod URLs was causing issues for QA since we cannot test the Twitter and Facebook cards until you push it to production, which isn't always ideal since it's too late at that point and we'd need more production pushes in order to verify tweaks.

Duplication of title/description strings doesn't seem DRY either. Not sure if we want to pass those into Handlebars as helpers (like we did for `availableLanguages`), not sure if that'd help w/ localization support or not since this would be server-side.